lms/static/scripts/frontend_apps/config.js

```diff
@@ -112,19 +112,19 @@ import { createContext } from 'preact';
  * The documentation for these properties lives in the `JSConfig` class in
  * the backend code.
  *
- * @typedef ConfigObject
+ * @typedef Config
```

Comment:
Can you revert this change. This type intentionally has an `Object` suffix to disambiguate it from the `Config` variable that this module exports.

Reply:
In my opinion, having a variable and  type with the same name is not a bad thing. Nevertheless, I have reverted.

Reply:
> In my opinion, having a variable and type with the same name is not a bad thing. 

I suppose this does happen with classes already, as you get both the class constructor value and the type derived from it. In that the case though the relationship between the type (constructor) and the value (instance type) is very obvious. In this case its a bit different than normal (the `Config` value is React context with a type of `ConfigObject`).